sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldNotBeIgnored.NUnit.cs

```diff
@@ -14,51 +17,95 @@ public void Foo1()
         }
 
         [Test]
-        [Microsoft.VisualStudio.TestTools.UnitTesting.Ignore] // This test is ignored because 'blah blah'
+        [NUnit.Framework.Ignore] // This test is ignored because 'blah blah'
```

Comment:
Previously all of these tests were using the MSTest _Ignore_ attribute, even though it does not work with NUnit tests, and NUnit has its own _Ignore_ attribute with the same semantics.
I've changed the remaining tests to use the NUnit attribute.